### PR TITLE
chore: add check commit message

### PR DESCRIPTION
**Pull request by anthony-redFox**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,5 +10,6 @@ before_script:
   - npm install -g grunt-cli
 
 script:
+  - commitlint-travis
```
Comment (johnjbarton) [inline .travis.yml:13]: What's the advantage of also triggering this in CI?  Won't husky solve this?

I'm not necessarily against this, but I would like all the karma-runner projects to be similar if possible.

Comment (anthony-redFox) [reply .travis.yml:13]: @johnjbarton 
husky - works on developer side and check could be skip by "git commit  --no-verify" that mean server side should be verify second time on PRs build.

I included `commitlint-travis` it is similar that you did in karma 
```
matrix:
  fast_finish: true
  include:
    - name: "Lint code and commit message format"
      node_js: "10"
      env: VALIDATE_COMMIT_MSG=true LINT=true
```
but it didn't execute additional thread with the same execution tests

Comment (anthony-redFox) [reply .travis.yml:13]: My suggestion to use @commitlint/travis-cli and have less hacks in .travis.yml file

Comment (anthony-redFox) [reply .travis.yml:13]: @johnjbarton  any suggestions?

Comment (johnjbarton) [reply .travis.yml:13]: Sorry got wrapped in trying to get the CI working on karma-runner.

I don't think `--no-verify` is too terrible for commit messages, worst case they don't trigger a release. But the check is also harmless if the commit is correct, so ok.

**Pull request merged**